hammurabi/config.py

```diff
@@ -182,19 +184,62 @@ def __merge_settings(self, loaded_settings: Dict[str, Any]) -> Settings:
 
         return Settings(**merge_result)
 
-    def load(self, file: Union[str, Path]):
+    def __get_fallback_repository(self) -> str:
+        """
+        Figure out the fallback owner/repository based on the remote url of the git repo.
+        :return: Returns the owner/repository pair
+        :rtype: str
+        """
+
+        repo_url: str = self.repo.remote().url
+
+        if re.match(r"^http(s)?://", repo_url):
```

Comment:
I think a simple `repo_url.startswith('http://') or repo_url.startswith('https://')` is slightly more readable here.

Reply:
Hm, maybe yes. The only concern I have against the `startswith` is that it's longer, which could make the code less readable on the long run. WDYT?

Reply:
I am fine with the regex as well. 😃👍